<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orphans</class>
 <widget class="QDialog" name="orphans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>569</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mantenimiento de paquetes huerfanos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recoverdrake.qrc">
    <normaloff>:/Imagenes/huerfano.png</normaloff>:/Imagenes/huerfano.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0" colspan="3">
      <widget class="QTableWidget" name="tableWidget">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Visor de registros de la base de datos.</string>
       </property>
       <property name="statusTip">
        <string>Visor de registros de la base de datos.</string>
       </property>
       <property name="whatsThis">
        <string>Visor de registros de la base de datos.</string>
       </property>
       <property name="autoScrollMargin">
        <number>26</number>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Marca</string>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/good.png</normaloff>:/Imagenes/good.png</iconset>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre de paquete huerfano</string>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignLeft|AlignVCenter</set>
        </property>
        <property name="foreground">
         <brush brushstyle="Dense4Pattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/rpm.png</normaloff>:/Imagenes/rpm.png</iconset>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>269</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Total Registros:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Numero de registros activos en la base de datos activa.</string>
       </property>
       <property name="statusTip">
        <string>Numero de registros activos en la base de datos activa.</string>
       </property>
       <property name="whatsThis">
        <string>Numero de registros activos en la base de datos activa.</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QTextEdit" name="textEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>234</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Nimbus Sans L Condensed</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Consola de procesos, donde se detalla al máximo lo que va procesando.</string>
       </property>
       <property name="statusTip">
        <string>Consola de procesos, donde se detalla al máximo lo que va procesando.</string>
       </property>
       <property name="whatsThis">
        <string>Consola de procesos, donde se detalla al máximo lo que va procesando.</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Nimbus Sans L Condensed'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:7pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QProgressBar" name="progressBar">
         <property name="toolTip">
          <string>Ratio de procesos realizados.</string>
         </property>
         <property name="statusTip">
          <string>Ratio de procesos realizados.</string>
         </property>
         <property name="whatsThis">
          <string>Ratio de procesos realizados.</string>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="maximumSize">
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Nos vamos..</string>
         </property>
         <property name="statusTip">
          <string>Nos vamos..</string>
         </property>
         <property name="whatsThis">
          <string>Nos vamos..</string>
         </property>
         <property name="text">
          <string>Salir</string>
         </property>
         <property name="icon">
          <iconset resource="recoverdrake.qrc">
           <normaloff>:/Imagenes/puerta.png</normaloff>:/Imagenes/puerta.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Acceso global</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="6" column="0">
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Actualizacion de los paquetes seleccionados.</string>
          </property>
          <property name="statusTip">
           <string>Deseleccion masiva de todos los registros.</string>
          </property>
          <property name="whatsThis">
           <string>Deseleccion masiva de todos los registros.</string>
          </property>
          <property name="accessibleName">
           <string>Deseleccion masiva de todos los registros.</string>
          </property>
          <property name="text">
           <string>Eliminar todos</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_6">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Seleccion masiva de todos los paquetes.</string>
          </property>
          <property name="statusTip">
           <string>Seleccion masiva de todos los paquetes.</string>
          </property>
          <property name="whatsThis">
           <string>Seleccion masiva de todos los paquetes.</string>
          </property>
          <property name="text">
           <string>Seleccionar todos</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/good.png</normaloff>:/Imagenes/good.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButton_7">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Deseleccion masiva de todos los paquetes.</string>
          </property>
          <property name="statusTip">
           <string>Deseleccion masiva de todos los paquetes.</string>
          </property>
          <property name="whatsThis">
           <string>Deseleccion masiva de todos los paquetes.</string>
          </property>
          <property name="text">
           <string>Deseleccionar todos</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_5">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Generar un fichero .lst tanto de la carga de otro fichero tras depurarlo, como de la generación de paquete huerfanos activos.</string>
          </property>
          <property name="statusTip">
           <string>Generar un fichero .lst tanto de la carga de otro fichero tras depurarlo, como de la generación de paquete huerfanos activos.</string>
          </property>
          <property name="whatsThis">
           <string>Generar un fichero .lst tanto de la carga de otro fichero tras depurarlo, como de la generación de paquete huerfanos activos.</string>
          </property>
          <property name="accessibleName">
           <string>Generar un fichero .lst tanto de la carga de otro fichero tras depurarlo, como de la generación de paquete huerfanos activos.</string>
          </property>
          <property name="text">
           <string>Generar fichero</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/filesave.png</normaloff>:/Imagenes/filesave.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="pushButton_8">
          <property name="toolTip">
           <string>Limpia la ventana de registros.</string>
          </property>
          <property name="statusTip">
           <string>Limpia la ventana de registros.</string>
          </property>
          <property name="whatsThis">
           <string>Limpia la ventana de registros.</string>
          </property>
          <property name="text">
           <string>Vaciar lista</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/escoba.png</normaloff>:/Imagenes/escoba.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_14">
          <property name="toolTip">
           <string>Comprobacion de los paquetes huefanos.</string>
          </property>
          <property name="statusTip">
           <string>Comprobacion de los paquetes huefanos.</string>
          </property>
          <property name="whatsThis">
           <string>Comprobacion de los paquetes huefanos.</string>
          </property>
          <property name="text">
           <string>Verificar huerfanos</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/lupaO.png</normaloff>:/Imagenes/lupaO.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Recuperar todos</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/undo.png</normaloff>:/Imagenes/undo.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Carga de fichero de una salvaguarda anterior para poder instalar, depurar o modificar.</string>
          </property>
          <property name="statusTip">
           <string>Carga de fichero de una salvaguarda anterior para poder instalar, depurar o modificar.</string>
          </property>
          <property name="whatsThis">
           <string>Carga de fichero de una salvaguarda anterior para poder instalar, depurar o modificar.</string>
          </property>
          <property name="text">
           <string>Cargar fichero</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/fileopen.png</normaloff>:/Imagenes/fileopen.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Acceso individual</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_10">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Marca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="statusTip">
           <string>Marca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="whatsThis">
           <string>Marca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="text">
           <string>Marcar seleccionado</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/good.png</normaloff>:/Imagenes/good.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_12">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Desmarca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="statusTip">
           <string>Desmarca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="whatsThis">
           <string>Desmarca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="text">
           <string>Desmarcar seleccionado</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_13">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Elimina todos los registros seleccionados.</string>
          </property>
          <property name="statusTip">
           <string>Elimina todos los registros seleccionados.</string>
          </property>
          <property name="whatsThis">
           <string>Elimina todos los registros seleccionados.</string>
          </property>
          <property name="text">
           <string>Eliminar seleccionado</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_11">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Quita de la lista el registro seleccionado.</string>
          </property>
          <property name="statusTip">
           <string>Quita de la lista el registro seleccionado.</string>
          </property>
          <property name="whatsThis">
           <string>Quita de la lista el registro seleccionado.</string>
          </property>
          <property name="text">
           <string>Quitar seleccionado</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recoverdrake.qrc"/>
 </resources>
 <connections/>
</ui>
